packages/dataviews/src/utils.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Helper util to sort items by text fields, when sorting is done client side.
+ *
+ * @param {Object}   params            Function params.
+ * @param {Object[]} params.items      Array of items to sort.
```

Comment:
items is called "data" in the dataviews component, should we keep the same name?

Reply:
I can do that.